app/Http/Controllers/Auth/EmailVerificationController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Foundation\Auth\EmailVerificationRequest;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\View\View;
+
+class EmailVerificationController extends Controller
+{
+    public function showNotice(): View
+    {
+        return view('auth.verify-email-notice');
+    }
+
+    public function resend(): RedirectResponse
+    {
+        $user = request()->user();
+
+        if ($user->hasVerifiedEmail()) {
+            return redirect()->route('dashboard')->with('message', 'Email already verified!');
+        }
+
+        request()->user()->sendEmailVerificationNotification();
```

Comment:
Change to `$user->sendEmailVerificationNotification();` since we already have the user